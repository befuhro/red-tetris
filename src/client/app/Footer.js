import React from 'react';
import WebFont from 'webfontloader';

import logo from '../assets/Le-101.png';

WebFont.load({
    google: {
        families: ['Permanent Marker']
    }
});

const footerStyle = {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    position: 'absolute',
    bottom: '0',
    width: '100%',
    height: '14.24vh',
    color: '#61dafb',
    backgroundImage: 'linear-gradient(rgba(53, 58, 73, 0.7), rgba(11, 29, 45, 1))',
};

const fontStyle = {
    fontFamily: 'Permanent Marker',
    fontSize: '3vh',
    fontWeight: '300',
    color: '#61dafb',
    marginRight: '2vh'
};

const imageStyle = {
    height: '10vh',
    marginLeft: '2vh'
};

function Footer() {
    return (
        <div style={footerStyle}>
            <p style={fontStyle}>befuhro & gmadec</p>
            <img src={logo} style={imageStyle} alt='le-101-logo'/>
        </div>
    );
}

export default Footer;