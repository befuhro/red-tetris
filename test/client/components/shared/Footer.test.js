import * as React from 'react';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

enzyme.configure({ adapter: new Adapter() });

import Footer from '../../../../src/client/components/shared/Footer';

describe('<Footer />', () => {
    let component;

    beforeAll(() => {
        component = enzyme.mount(<Footer/>);
    });

    it('render basic', () => {
        expect(component.html()).toMatchSnapshot();
    });
});
