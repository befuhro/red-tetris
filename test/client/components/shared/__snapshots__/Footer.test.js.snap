// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render basic 1`] = `"<div style=\\"display: flex; justify-content: center; align-items: center; position: absolute; bottom: 0px; width: 100%; height: 14.24vh; color: rgb(97, 218, 251);\\"><p style=\\"font-family: Permanent Marker; font-size: 3vh; font-weight: 300; color: rgb(97, 218, 251); margin-right: 2vh;\\">befuhro &amp; eruaud</p><img src=\\"\\" style=\\"height: 10vh; margin-left: 2vh;\\" alt=\\"le-101-logo\\"></div>"`;
